docker/0.90-2/resources/mms/config.properties

```diff
@@ -1,4 +1,4 @@
-vmargs=-Xmx128m -XX:-UseLargePages -XX:+UseG1GC -XX:MaxMetaspaceSize=32M -XX:MaxDirectMemorySize=10m -XX:+ExitOnOutOfMemoryError
+vmargs=-Xmx128m -XX:-UseLargePages -XX:+UseG1GC -XX:MaxMetaspaceSize=32M -XX:MaxDirectMemorySize=1G -XX:+ExitOnOutOfMemoryError
```

Comment:
Have you done any measurement? Why 1G and, say, not 100M?

Reply:
1GB came from a quick napkin calculation. The buffer is shared across the JVM, and if each workers was concurrently receiving 20MB payloads that would allow roughly 50 workers to work in parallel. The largest number of CPU's a sagemaker instance has is 96 based on https://aws.amazon.com/sagemaker/pricing/instance-types/ 

This is obviously not a perfect solution. I think if we had bandwidth i would want to test calculating num_cpu * max_payload_size and setting that as a env var that can be parsed by MMS to use in the config file. This change is just to unblock Flintstone to use 20MB payloads